docs/content/guides/dev/building-and-deploying-gloo.md

```diff
@@ -9,9 +9,6 @@ You can build and deploy Gloo Gateway Open Source from the source code.
 
 1. Follow the [setup guide]({{% versioned_link_path fromRoot="/guides/dev/setting-up-dev-environment/" %}}) to clone the Gloo Gateway repository and install the project dependencies.
 2. In your terminal, navigate to the Gloo Gateway project.
```

Comment:
just curious why this was removed?

Reply:
`GOPATH` hasn't been relevant since Go mod was introduced years ago. It also removed the need to organize projects with the same directory structure as the package's path.

Reply:
https://go.dev/wiki/GOPATH#deprecating-and-removing-gopath-development-mode

Developers are free to organize their repo directories however they want.